src/nri/metricsProcesor.go

```diff
@@ -18,34 +18,36 @@ import (
 	dto "github.com/prometheus/client_model/go"
 )
 
+const (
+	entityNamePrefix = "win_service"
```

Comment:
why not `WIN_SERVICE` like the entity type itself ?

Reply:
What do you mean, in capital letters?

Reply:
Ok, I set it up in capital letters